code/datums/gamemodes/game_mode.dm

```diff
@@ -338,6 +340,7 @@ Sensors indicate [numXenosShip || "no"] unknown lifeform signature[numXenosShip
 
 	for(var/i in GLOB.player_list)
 		var/mob/M = i
+		grant_eord_respawn(null, M)
```

Comment:
isnt this done on COMSIG_GLOB_MOB_LOGIN in line 332

Reply:
Yes but if you don't enter a new mob (ie you were already a ghost) you never got the verb.